apps/cyberstorm-storybook/stories/components/TextInput.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { TextInput } from "@thunderstore/cyberstorm";
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSkull } from "@fortawesome/free-solid-svg-icons";
+
+const meta = {
+  title: "Cyberstorm/Components/TextInput",
+  component: TextInput,
+} as ComponentMeta<typeof TextInput>;
+
+const defaultArgs = {
```

Comment:
Seems I always find something to say about `defaultArgs`. This time the question is: why is it defined up here when it's used way down there?

Reply:
My best guess is; I've just had it there and didn't occur to me to move it closer to where it's used 🤷